sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/AnalyzeSentimentAsync.java

```diff
@@ -19,8 +20,8 @@
     public static void main(String[] args) {
         // Instantiate a client that will be used to call the service.
         TextAnalyticsAsyncClient client = new TextAnalyticsClientBuilder()
-            .subscriptionKey("{subscription_key}")
-            .endpoint("https://{servicename}.cognitiveservices.azure.com/")
+            .subscriptionKey(new TextAnalyticsSubscriptionKeyCredential("{subscription_key}"))
+            .endpoint("{endpoint}")
```

Comment:
Why the change to `{endpoint}`? It is more clear what the string should look like with the last change.

Reply:
Based on what Mariana point out here: https://github.com/Azure/azure-sdk-for-java/pull/7596#discussion_r369861889 

The subscription key could apply to both Cognitive Subscription Key or Text Analytics Subscription key.